config/schedule.rb

```diff
@@ -106,6 +106,9 @@
   every 1.day, at: '6:01 am' do
     rake "health:enrollments_and_eligibility"
   end
+  every 1.day, at: '8:01 am' do
+    rake "health:queue_scheduled_documents"
+  end
```

Comment:
Do we need a specific time for this?  If not, I'd be inclined to roll it into
```
every 1.day, at: '6:01 am' do
     rake "health:enrollments_and_eligibility"
   end
```

Reply:
For the E/D report, it is affected by the 834 which is processed asynchronously, but since it is a general process, I just scheduled it for a different time... I don't really know if 8:00 is different enough, or maybe it should be run earlier than the morning processing?